cves/CVE-2017-15706.yml

```diff
@@ -103,9 +115,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: no
```

Comment:
I would suggest changing areas in which you wrote "no", to be "false" to match the instructions.

Reply:
I would also recommend writing out an answer to the "answer" section. Was there other unit tests present in the system, just not for this particular vulnerability?

Reply:
resolved.